templates/error/piskel-unauthorized-action.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+
+{% block title %}Insufficient permissions !{% endblock %}
+
+{% block css %}
+  <link rel="stylesheet" type="text/css" href="/static/css/piskel-app-piskel.css"/>
+{% endblock %}
+
+{% block main %}
+  <div class="piskel-error-container">
+    <h2 class="piskel-error-title pskl-important font-pixel">Sorry !</h2>
+    <h3 class="piskel-error-details font-pixel">Insufficient permissions</h3>
+    {% if not is_logged_in %}
+        Did you forgot to <a href="/auth/google">login</a>?<br>
+    {% else %}
+        Make sure the piskel you are trying to perform action on is owned by you.
```

Comment:
nit: "Make sure the piskel you are trying to perform action on" -> "Make sure the piskel you are trying to perform this/an action on" or "Make sure the piskel on which you are trying to perform this action"